scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -355,11 +355,17 @@ <h6>Phenotype terms (HPO):</h6>
     {% endif %}
     </div>
     <div class="col-3">
-    {% if variant.is_mitochondrial and case.mt_bams or case.bam_files %}
```

Comment:
Perhaps to keep it simple one could keep the original if and else (which were working) and in the case of else put the code for a button which is deactivated and has the tooltip. Because honestly it would be easier to maintain..

Reply:
> Sure, though that will be a bit redundant. I would prefer not to keep this statement, as it is very hard to read. Jinja dont really make there operator priority super clear, and it lacks parenthesis to disambiguate `(variant.is_mitochondrial and case.mt_bams) or case.bam_files` from `variant.is_mitochondrial and (case.mt_bams or case.bam_files)`

You can also keep it as it was:

else: "No alignments available"